src/backport.ts

```diff
@@ -104,7 +104,7 @@ const backportOnce = async ({
     await git("checkout", "-b", head);
     try {
       try {
-        await git("cherry-pick", "-n", commitToBackport);
+        await git("cherry-pick", "-m", "1", "-n", commitToBackport);
```

Comment:
```suggestion
        await git("cherry-pick", "-m1", "-n", commitToBackport);
```
this makes it a bit clearer